### ISSUE TITLE
Navigator Error

### ISSUE THREAD

**Issue by cameronsr** [ContinuumIO/anaconda-issues#2648]
## Main error
The following errors occurred:
## Traceback
```
CondaHTTPError: HTTP None None for url <https://repo.continuum.io/pkgs/free/win-64/repodata.json.bz2>
Elapsed: None

An HTTP error occurred when trying to retrieve this URL.
HTTP errors are often intermittent, and a simple retry will get you on your way.
SSLError(SSLError(SSLError("bad handshake: Error([('SSL routines', 'ssl3_get_server_certificate', 'certificate verify failed')],)",),),)

```
## System information
```
python: 3.6.1
language: en_US
os: Windows;10;10.0.10586;AMD64;Intel64 Family 6 Model 58 Stepping 9, GenuineIntel
version: 1.6.2
platform: win-64
qt: 5.6.2
pyqt: 5.6.0
conda: 4.3.21
```

**Comment by goanpeca** [comment]
Closing as duplicate of #1932

Please update to the latest version of Navigator.

On Navigator click on the update button on the top right of the interface or
on the terminal type

```
$ conda update anaconda-navigator
```

**Comment by cameronsr** [comment]
Issue traced to Preferences in Navigator. Anaconda API domain setting (currently https://api.anaconda.org) is flagged as invalid URL. Because of this, not possible to update from terminal. This setting was auto configured on initial install. Has URL changed, and if so what is the new API domain URL? Cannot find a reference to this setting in online documentation.

**Comment by WolfDSCoder** [comment]
I'm having the same issue as @cameronsr .

Anaconda API domain setting (currently https://api.anaconda.org) is flagged as an invalid URL.

Also, I can't create a new environment in Anaconda. I get the same error as originally posted in this thread. Is there a fix for this?

Thank you.


CondaHTTPError: HTTP None None for url <https://repo.continuum.io/pkgs/free/win-64/repodata.json.bz2>
Elapsed: None

An HTTP error occurred when trying to retrieve this URL.
HTTP errors are often intermittent, and a simple retry will get you on your way.
SSLError(SSLError(SSLError("bad handshake: Error([('SSL routines', 'ssl3_get_server_certificate', 'certificate verify failed')],)",),),)

**Comment by cameronsr** [comment]
@WolfDSCoder I received a message after my last comment stating this was fixed. In the end I wound up having to reinstall Anaconda after a clean uninstall. No problems since, but haven't yet tried to update to the new Python distro that just came out. Will proceed later today and note my results on this thread.

**Comment by WolfDSCoder** [comment]
I just tried to use conda install anaconda-clean from the Anaconda Prompt, but get the same CondaHTTPError.

I also tried uninstalling Anaconda from Win10 Programs and Features, then reinstalling Anaconda from a fresh download. The same issues happened. Can't create an environment. Can't update the preferences for the Anaconda API domain setting.

Not sure what to do next. Thank you for helping.

**Comment by cameronsr** [comment]
WolfDSCoder - I would open a new issue referencing this one since they've
closed it.

On Jul 17, 2017 15:13, "WolfDSCoder" <notifications@github.com> wrote:

I just tried to use conda install anaconda-clean from the Anaconda Prompt,
but get the same CondaHTTPError.

I also tried uninstalling Anaconda from Win10 Programs and Features, then
reinstalling Anaconda from a fresh download. The same issues happened.
Can't create an environment. Can't update the preferences for the Anaconda
API domain setting.

Not sure what to do next. Thank you for helping.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/ContinuumIO/anaconda-issues/issues/2648#issuecomment-315900518>,
or mute the thread
